src/Microsoft.ML.Data/DataLoadSave/EstimatorChain.cs

```diff
@@ -72,7 +72,14 @@ public EstimatorChain<TNewTrans> Append<TNewTrans>(IEstimator<TNewTrans> estimat
             where TNewTrans : class, ITransformer
         {
             Contracts.CheckValue(estimator, nameof(estimator));
+
             return new EstimatorChain<TNewTrans>(_estimators.AppendElement(estimator), _scopes.AppendElement(scope));
         }
+
+        public EstimatorChain<TNewTrans> Append<TNewTrans>()
```

Comment:
>Append [](start = 41, length = 6)

Why is this needed? And why does it create a new class?

Reply:
Actually, this looks like a crude way to re-map the existing chain 'that ends with Foo' to another chain 'that ends with Bar'. I don't think this functionality is good, or useful in this case.

---
In reply to: [229384750](https://github.com/dotnet/machinelearning/pull/1426#discussion_r229384750) [](ancestors = 229384750)